core/java/src/org/openda/blackbox/config/IoObjectToDataObjectDictionary.java

```diff
@@ -0,0 +1,30 @@
+package org.openda.blackbox.config;
+
+import java.util.Collections;
+import java.util.HashMap;
+import java.util.Map;
+
+public class IoObjectToDataObjectDictionary {
```

Comment:
Codacy found an issue: [The utility class name 'IoObjectToDataObjectDictionary' doesn't match '[A-Z][a-zA-Z0-9]+(Utils?|Helper|Constants)'](https://app.codacy.com/gh/OpenDA-Association/OpenDA/pullRequest?prid=5102854)